learners/setup.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Setup
+---
+
+All hands-on work will likely be done on an HPC cluster with Chapel installed. Alternatively, you can run
+Chapel via Docker on your computer.
+
```

Comment:
Populate this file with Chapel installation instructions.

This lesson does not require a cluster.

Reply:
I added Chapel installation instructions for Mac and Linux and a placeholder for Windows. I don't have access to Windows, so I don't really know how to run Docker on Windows, apart from quick googling. I suspect the Docker option will provide the best scaling results, if you don't already have the WSL enabled.